pkg/config/constants.go

```diff
@@ -18,12 +18,22 @@ package config
 
 // Add a field here if it never changes, if it changes over time, put it to settings.go
 const (
-	APIGroup    = "csi.ibm.com"
-	APIVersion  = "v1"
-	Name        = "ibm-block-csi-operator"
-	DriverName  = "block.csi.ibm.com"
-	ProductName = "ibm-block-csi-driver"
-	Masterlabel = "node-role.kubernetes.io/master"
+	APIGroup                  = "csi.ibm.com"
+	APIVersion                = "v1"
+	Name                      = "ibm-block-csi-operator"
+	DriverName                = "block.csi.ibm.com"
+	ProductName               = "ibm-block-csi-driver"
+	Masterlabel               = "node-role.kubernetes.io/master"
```

Comment:
@oriyarde do we want to remove all unused consts?

Reply:
sure

Reply:
so @matancarmeli7 please remove all the unused consts here

Reply:
done